apps/entity-service/src/entities/dto/site-report.dto.ts

```diff
@@ -0,0 +1,302 @@
+import { SiteReport } from "@terramatch-microservices/database/entities";
+import { EntityDto, AdditionalProps } from "./entity.dto";
+import { pickApiProperties } from "@terramatch-microservices/common/dto/json-api-attributes";
+import { JsonApiDto } from "@terramatch-microservices/common/decorators/json-api-dto.decorator";
+import { ApiProperty } from "@nestjs/swagger";
+import { MediaDto } from "./media.dto";
+import { User } from "@sentry/nestjs";
+import { ProjectReportLightDto } from "./project-report.dto";
+
+@JsonApiDto({ type: "siteReports" })
+export class SiteReportLightDto extends EntityDto {
+  constructor(siteReport?: SiteReport, props?: AdditionalSiteReportLightProps) {
+    super();
+    if (siteReport != null) {
+      this.populate(SiteReportLightDto, {
+        ...pickApiProperties(siteReport, SiteReportLightDto),
+        lightResource: true,
+        // these two are untyped and marked optional in the base model.
+        createdAt: siteReport.createdAt as Date,
+        updatedAt: siteReport.createdAt as Date,
+        ...props
+      });
+    }
+  }
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated site name"
+  })
+  siteName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated site uuid"
+  })
+  siteUuid: string | null;
+
+  @ApiProperty()
+  frameworkKey: string | null;
+
+  @ApiProperty()
+  frameworkUuid: string | null;
+
+  @ApiProperty()
+  status: string;
+
+  @ApiProperty()
+  updateRequestStatus: string;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project name"
+  })
+  projectName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project uuid"
+  })
+  projectUuid: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation name"
+  })
+  organisationName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation uuid"
+  })
+  organisationUuid: string | null;
+
+  @ApiProperty()
+  updatedAt: Date;
+
+  @ApiProperty({ nullable: true })
+  submittedAt: Date | null;
+
+  @ApiProperty({ nullable: true })
+  taskId: number | null;
```

Comment:
I think we want to be sending task UUIDs to the client instead of the integer ID.